aten/src/THC/generic/THCTensorMode.cu

```diff
@@ -1,3 +1,6 @@
+#if defined(__HIP_PLATFORM_HCC__)
+#include <thrust/iterator/constant_iterator.h>
```

Comment:
Why is here conditionally included but not in `aten/src/ATen/native/cuda/EmbeddingBag.cu`?

Reply:
This one had no thrust includes - EmbeddingBag already had a bunch so it looked like an oversight. More than happy to remove the conditionals here. Shall I?